src/main/java/com/hubspot/jinjava/el/ext/ExtendedParser.java

```diff
@@ -255,7 +255,8 @@ protected AstDict dict() throws ScanException, ParseException {
       }
     }
 
-    if (!getToken().getImage().equals("}")) {
+    Scanner.Token nextToken = getToken();
+    if (nextToken == null || nextToken.getImage() == null || !nextToken.equals("}")) {
```

Comment:
```suggestion
    if (nextToken == null || !"}".equals(nextToken.getImage())) {
```
Typo, you're checking `nextToken.equals` rather than `nextToken.getImage().equals`